modules/check_in/app/sidekiq/check_in/travel_claim_base_job.rb

```diff
@@ -53,18 +53,38 @@ def send_notification(opts = {})
       begin
         va_notify_send_sms(opts)
       rescue => e
-        retry_attempt += 1
-        if retry_attempt <= MAX_RETRIES
-          log_send_sms_failure(retry_attempt)
-
-          sleep(retry_attempt * 2) # Backoff: 2s, 4s, 6s
-          retry
-        end
-        handle_error(e, opts)
+        log_send_sms_failure(retry_attempt)
```

Comment:
This is missing the increment for retry_attempt and MAX_RETRIES is still set to 3 above. That said I don't know if this works. It's unclear to me if the increment will work the same since the retry keyword is no longer in use. It might be good to have a test for the error messages that state the retry attempt if possible.